ui/src/overview.tsx

```diff
@@ -296,7 +300,10 @@ function ParticipantCriteria(props: {
       groups: [
         {
           id: props.group.id,
-          kind: tanagra.GroupKindEnum.Included,
+          filter: {
```

Comment:
Is this the default group filter?

Reply:
Not quite but that would be a bit simpler and get the same result. Done.